src/App.tsx

```diff
@@ -1,9 +1,58 @@
+import { useEffect } from "react";
 import styles from "./App.module.css";
+import {faker} from "@faker-js/faker";
 
-const App = () => (
+interface Flight {
+  flightNumber: string;
+  origin: string;
+  destination: string;
+  departureTime: Date;
+  arrivalTime: Date;
+  flightDuration: number;
+}
+
+
+
+const App = () => {
+
+  const [allFlights, setFlights] = useState(0);
```

Comment:
Let's initialize our flights as type flight with a default value of an empty array!
```suggestion
  const [allFlights, setFlights] = useState<Flight[]>([]);
```